test/checks/mobile/target-size.js

```diff
@@ -411,4 +410,19 @@ describe('target-size tests', function () {
       '#obscurer2'
     ]);
   });
+
+  it('ignores descendants of the target that are in shadow dom', () => {
+    fixture.innerHTML =
```

Comment:
`shadowCheckSetup` assumes that the target is either the shadow root node or inside the shadow content. This isn't true for the DOM I was testing and I didn't want to try to update the function in this PR to account for content which uses both.